modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -107,16 +107,18 @@ NOTE: With standard BYOC clusters, Redpanda manages security policies and resour
 
 == Cluster tiers
 
-When you create a cluster, you select your throughput tier. For example, the following table lists the current AWS tier limits. Legacy tiers may have different limits. For more information, or for the current GCP limits, contact https://support.redpanda.com/hc/en-us/requests/new[support^]. 
+When you create a cluster, you select your throughput tier. 
 
-|===
-| | Maximum ingress | Maximum egress | Maximum partitions | Maximum connections
+The following table lists current AWS and GCP tier limits. Legacy tiers may have different limits. For more information, contact https://support.redpanda.com/hc/en-us/requests/new[support^]. 
+
+|=== 
+| Tier | Maximum ingress | Maximum egress | Maximum partitions | Maximum connections
 
-| *Tier 1* | 20 MBps | 60 MBps | 1,000 | 1,500
-| *Tier 2* | 50 MBps | 150 MBps | 2,800 | 3,750
-| *Tier 3* | 100 MBps | 200 MBps | 5,600 | 7,500
-| *Tier 4* | 200 MBps | 400 MBps | 11,200 | 15,000
-| *Tier 5* | 400 MBps | 800 MBps | 22,800 | 30,000
+| Tier 1 | 20 MBps | 60 MBps | 1,000 | 1,500
+| Tier 2 | 50 MBps | 150 MBps | 2,800 | 3,750
+| Tier 3 | 100 MBps | 200 MBps | 5,600 | 7,500
+| Tier 4 | 200 MBps | 400 MBps | 11,300 | 15,000
```

Comment:
I don't think this should be 11,300, should be 11,200

Reply:
confirmed with Tristan that limit is 11,300